addon/mixins/application-route-mixin.js

```diff
@@ -58,6 +58,10 @@ export default Mixin.create({
     return fastboot ? fastboot.get('isFastBoot') : false;
   }),
 
+  _cookies: computed(function() {
+    return getOwner(this).lookup('service:cookies');
+  }),
```

Comment:
Yeah, that makes testing easier for sure. There should be examples in the code base where the container is mocked though.